lib/main_common.pm

```diff
@@ -1167,7 +1167,7 @@ sub load_x11tests {
         loadtest "x11/dolphin";
     }
     # SLES4SAP default installation does not configure snapshots
-    if (snapper_is_applicable() and !is_sles4sap()) {
+    if (snapper_is_applicable() and !is_updates_tests() and !is_sles4sap()) {
```

Comment:
It is not obvious from the code why we should exclude the snapshot tests for maintenance tests. So I suggest to either do snapshot tests in maintenance tests which I think makes sense or you put the `!is_updates_tests` with a comment why we do not want to conduct snapper tests in maintenance tests within `snapper_is_applicable`

Reply:
actually we do want these tests for maintenance. But not for leap 42.*